static/js/lib/location.js

```diff
@@ -46,10 +46,11 @@ export const nameToStateCode = (country: string, name: string) =>  {
   if (!R.isEmpty(statecode)) {
     return statecode['code'];
   } else {
-    let subcodes = R.map(function(key) {
-      return {"code": key, "name": iso3166.data[country]["sub"][key]["name"]};
-    }, Object.keys(iso3166.data[country]["sub"]));
-    let fuse = new Fuse(subcodes, {keys: ["name"], threshold:0.4, tokenize: true});
+    const getSubcodes = (key: string) => (
+      {"code": key, "name": iso3166.data[country]["sub"][key]["name"]}
+    );
+    const subcodes = R.map(getSubcodes, Object.keys(iso3166.data[country]["sub"]));
+    const fuse = new Fuse(subcodes, {keys: ["name"], threshold:0.4, tokenize: true});
```

Comment:
sorry, I wasn't clear! I was proposing moving it out of the `nameToStateCode` function entirely, something like:

```js
const getSubCodes = country => R.map(
  key => ({ code: key, name: iso3166.data[country].sub.key["name"] }),
  Object.keys(iso3166.data[country].sub)
);
```

Then when you're creating your `Fuse` object you could just call `getSubcodes(country)` inline.